backend/de.metas.business/src/main/java/de/metas/shipping/api/impl/ShipperTransportationDAO.java

```diff
@@ -177,4 +180,27 @@ public ImmutableList<OrderId> retrieveOrderIds(@NonNull final ShipperTransportat
 				.listDistinct(I_M_ShippingPackage.COLUMNNAME_C_Order_ID, OrderId.class);
 
 	}
+
+	@Override
+	public Collection<ShipperTransportationId> getTransportationOrderIdsAssignedToOrders(@NonNull final IQueryFilter<I_C_Order> orderQueryFilter)
+	{
+		return queryBL.createQueryBuilder(I_C_Order.class)
```

Comment:
pls avoid querying C_Order in this DAO.
OrderDAO shall be responsible for that.